package.json

```diff
@@ -55,9 +55,9 @@
     "transpile:clean": "rm -rf ./dist",
     "transpile:js": "BABEL_ENV=dev babel packages --ignore ./packages/bpk-stylesheets --out-dir dist --extensions \".ts,.tsx,.js,.jsx\" --config-file ./babel.config.js",
     "transpile:dts": "tsc --project tsconfig.declaration.json",
-    "transpile:imports": "node scripts/transpilation/transform-js-scss-css-imports.js",
     "transpile:copy-css": "node scripts/transpilation/copy-css.js",
     "transpile:copy-utils": "node scripts/transpilation/copy-utils.js",
+    "transpile:imports": "node scripts/transpilation/transform-js-scss-css-imports.js",
```

Comment:
Changing the order so that we will capture `bpk-stylesheets` otherwise the import change would happen before bpk-stylesheets was ready